app/gui/src/project-view/components/ActionButton.vue

```diff
@@ -11,22 +10,13 @@ const action = computed(() =>
 )
 const descriptionWithShortcut = computed(() =>
   action.value.shortcut ?
-    `${toValue(action.value.description)} (${toValue(action.value.shortcut)})`
+    `${toValue(action.value.description)} (${toValue(action.value.shortcut?.humanReadable)})`
   : toValue(action.value.description),
 )
 </script>
 
 <template>
-  <ToggleIcon
-    v-if="action.toggled != null"
-    :modelValue="toValue(action.toggled)"
-    :icon="toValue(action.icon)"
-    :disabled="toValue(action.disabled)"
-    :title="descriptionWithShortcut"
-    @click.stop="action.action"
-  />
```

Comment:
This effectively hides the information from toggle-able actions if they are toggled on or off.

It affects _all_ action buttons, like color picker. Currently, it has more vivid color when toggled on:

https://github.com/user-attachments/assets/d19c863a-36cf-42a3-91a2-8cd13cdb8697

While this one is not very important for the user, but the actions are meant to be very generic, and I guess the lack of feedback that the action is toggleable is concerning.

Also, I think the color-picker button itself does _not_ look disabled; perhaps we could just use the same tone in visualization button?

![image](https://github.com/user-attachments/assets/6e498399-97b9-4dc3-b481-3e991e58224f)

Reply:
Yes I thought about this. The counter argument is that in both of these cases the user can see that it is toggled on as there is a new pane that is opened.

But yes the other option is too make untoggled the same color as the other buttons and then do something else for toggled on (perhaps we can go even darker)

Reply:
Or this for toggled on?
![image](https://github.com/user-attachments/assets/f547320d-b042-4b73-92f6-3f8ed2decb6f)

Reply:
Yes, we can just make `ToggleButton` has different behavior (like the highlight instead of icon color change).